extensions/measurement-tracking/src/_shared/setCornerstoneMeasurementActive.js

```diff
@@ -0,0 +1,32 @@
+import cornerstone from 'cornerstone-core';
```

Comment:
This would be a weird command, but it could be a command in the Cornerstone extension.

Could also add it to `lib` exports in cornerstone-tools, even though it's a bit of workaround